### PR TITLE
[ticket/10165] Add a "Send test email" feature

### PR DESCRIPTION
**Pull request by Zoddo**
https://tracker.phpbb.com/browse/PHPBB3-10165

### CODE REVIEW

File: phpBB/language/en/acp/board.php [hunk 1]
```diff
@@ -574,6 +576,7 @@
 	'SMTP_SETTINGS'					=> 'SMTP settings',
 	'SMTP_USERNAME'					=> 'SMTP username',
 	'SMTP_USERNAME_EXPLAIN'			=> 'Only enter a username if your SMTP server requires it.',
+	'TEST_EMAIL_SENT'				=> 'The testing email has been sent.',
```
Comment (VSEphpbb) [inline phpBB/language/en/acp/board.php]: "testing" should just be "test" in all these new lang keys.

File: phpBB/language/en/email/test.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Subject: phpBB is correctly configured to send emails
+
+Hello {USERNAME},
+
+Congratulations, if you receive this email, phpBB is correctly configured to send emails.
```
Comment (VSEphpbb) [inline phpBB/language/en/email/test.txt]: Should be "received"

**Comment by Nicofuma** [comment]
looks good

File: phpBB/includes/acp/acp_board.php [hunk 3]
```diff
@@ -631,6 +632,27 @@ function main($id, $mode)
 			}
 		}
 
+		if ($mode == 'email' && isset($_POST['send_test_email']))
```
Comment (marc1706) [inline phpBB/includes/acp/acp_board.php]: Please use ```$request->is_set_post()```

File: phpBB/language/en/email/test.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Subject: phpBB is correctly configured to send emails
+
+Hello {USERNAME},
+
+Congratulations, if you received this email, phpBB is correctly configured to send emails.
```
Comment (marc1706) [inline phpBB/language/en/email/test.txt]: Congratulations. If you received this ...

File: phpBB/language/en/email/test.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+If you need any help, you can visit phpBB.com's community board - https://www.phpbb.com/community/
```
Comment (marc1706) [inline phpBB/language/en/email/test.txt]: In case you require assistance, please visit the phpBB support forums - https://www.phpbb.com/community/

**Pull request merged**